<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Центральный сервер</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="textLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Объекты</string>
    </property>
    <addaction name="boreAction"/>
   </widget>
   <widget class="QMenu" name="menuGSM">
    <property name="title">
     <string>GSM</string>
    </property>
    <addaction name="smsAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuGSM"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="boreAction">
   <property name="text">
    <string>Скважины</string>
   </property>
  </action>
  <action name="smsAction">
   <property name="text">
    <string>Отправить сообщение</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
